packaging/torchvision/meta.yaml

```diff
@@ -10,7 +10,7 @@ requirements:
   build:
     - {{ compiler('c') }} # [win]
     - libpng
-    - jpeg
+    - libjpeg-turbo
```

Comment:
@malfet @seemethere ,
I'm trying to package torchvision against libjpeg-turbo instead of libjpeg.

libjpeg-turbo is available on conda-forge [for all platforms](https://anaconda.org/conda-forge/libjpeg-turbo), but on the default `anaconda` channel it is [only available](https://anaconda.org/anaconda/libjpeg-turbo) on Win and OSX, not Linux.

This causes the `binary_linux_conda` jobs to [fail](https://app.circleci.com/pipelines/github/pytorch/vision/17335/workflows/e028f369-462a-4818-944b-acae268c70c2/jobs/1403140) (the rest are fine).

Would you know how we could rely on `conda-forge` here?

Reply:
I think I fixed that in f81da49